02-FlexBox/assignment/css/_components.content.scss

```diff
@@ -8,6 +17,63 @@ html {
     box-sizing: inherit;
 }
 
+/* CSS grid to match design spec link
+auto row height on mobile */
+.grid-container {
+    display: grid;
+    grid-template-rows: 149fr 81fr 649fr 152fr;
```

Comment:
Great! One improvement would be make the purple and orange boxes go full width on mobile for a bit of a better experience on mobile.